bundler.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+import subprocess
+import argparse
+import os
+import libroll as lib
+import deps
+from processes import PROCESS_MGR
+
+####################################################################################################
+# CONSTANTS
+
+# Minimum Go version required by the Stackup bundler.
+GO_VERSION = "1.19"
+
+####################################################################################################
+# ARGUMENT PARSING
+
+# Store the parsed arguments here.
+global args
+
+parser = argparse.ArgumentParser(
+    description="Helps you spin up an op-stack rollup.")
+
+subparsers = parser.add_subparsers(
+    title="commands",
+    dest="command",
+    metavar="<command>")
+
+subparsers.add_parser(
+    "start",
+    help="start an ERC4337 bundler")
+
+subparsers.add_parser(
+    "clean",
+    help="cleanup bundler processes")
+
+parser.add_argument(
+    "--no-ansi-esc",
+    help="disable ANSI escape codes for terminal manipulation",
+    default=True,
+    dest="use_ansi_esc",
+    action="store_false")
+
+####################################################################################################
+# SETUP
+
+def start():
+    setup_4337_contracts()
+    setup_stackup_bundler()
+    setup_paymaster()
+
+# --------------------------------------------------------------------------------------------------
+
+def setup_4337_contracts():
+    github_url = "https://github.com/0xFableOrg/account-abstraction.git"
```

Comment:
We make quite some change in the deployment scripts. When eth foundation release a new version, these deployment scripts will likely need to change too. Personally, I think it's easier to just make these changes directly to the deployment scripts of the fork, rather than making a bunch of changes in the patch file. But I don't have a really strong opinion, so can also do it the other way as you mentioned if preferred.